apycula/tiled_fuzzer.py

```diff
@@ -31,6 +31,12 @@
 device = sys.argv[1]
 
 params = {
+    "GW1NS-2": {
+        "package": "QN48",
```

Comment:
Usually it's best to take the package with the most pins here, this speeds up fuzzing.

Reply:
Done